_docs/for-contributors/2017-01-01-building-from-source.md

```diff
@@ -39,7 +47,31 @@ Switch to the latest release branch, for example, `cardano-sl-1.0`:
 
     $ git checkout cardano-sl-1.0
 
-Then enter `nix-shell`:
+## Pure Nix build mode (recommended)
+
+Two simple steps:
+
+1.  To employ the signed IOHK build cache:
+
+        $ sudo mkdir -p /etc/nix
+        $ sudo vi /etc/nix/nix.conf       # ..or any other editor, if you prefer
+
+    ..and then add two following lines:
+
+        $ binary-caches             = https://cache.nixos.org https://hydra.iohk.io
+        $ binary-caches-public-keys = hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ=
+
+2.  Actually building Cardano SL (or, most likely, simply obtaining it from the IOHK's binary caches) can be performed by:
+
+        $ nix-build -A cardano-sl-static --cores 0 --jobs 2 --no-build-output --out-link cardano-sl-1.0
+
+    The build output directory will be symlinked as `cardano-sl-1.0` (as specified by the command).
```

Comment:
This should explain what binaries are useful and how to run them from the output link.

Also we need to document what other attributes are useful (cardano-sl-tools, cardano-sl-wallet).

Reply:
@domenkozar, sounds like a good idea!

When should we do it?